src/4_initialization.js

```diff
@@ -35,7 +35,7 @@ else if (typeof exports === 'object') {
 if (!TITANIUM_BUILD) {
 	// Always make a global.
 	if (window) {
-		window['branch'] = branch_instance;
+		window.branch = branch_instance;
```

Comment:
most of the places we do this are actually intentional: see "Using string names to refer to object properties" in https://developers.google.com/closure/compiler/docs/limitations#implications-of-global-variable-function-and-property-renaming